_posts/2017/2017-03-22-microxchg-2017-microservices-conference.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "microXchg 2017 - Microservices conference in Berlin"
+date: "2017-02-22 12:02:22"
+image: blog-header/microxchg-2017.jpg
+categories: events
+authors: ["Benjamin"]
+---
+
+# MicroXchg 2017 - Microservices conference in Berlin
+
+At mid of February the [microXchg conference](http://microxchg.io) took place in Berlin.
+It is the biggest two day event in Europe that focuses on Microservices - an architectural design of software systems that has generated a lot of traction within the last years as it is capable of transforming SOA and monolith systems.
+
+## Kickoff Conference Panel - on evening before
+
+One day before the [Microservices Meetup Berlin](https://www.meetup.com/Microservices-Meetup-Berlin/) held a [conference panel](https://www.meetup.com/Microservices-Meetup-Berlin/events/237309001/) event with a plenum discussions.
```

Comment:
One day before the main conference [Microservices Meetup Berlin](https://www.meetup.com/Microservices-Meetup-Berlin/) there was a [conference panel](https://www.meetup.com/Microservices-Meetup-Berlin/events/237309001/) event with a plenum discussion.